internal/bminventory/inventory.go

```diff
@@ -4114,7 +4120,7 @@ func (b *bareMetalInventory) RegisterInfraEnvInternal(
 
 func (b *bareMetalInventory) validateInfraEnvCreateParams(ctx context.Context, params installer.RegisterInfraEnvParams, cluster *common.Cluster) error {
 	var err error
-	if cluster != nil && cluster.CPUArchitecture != "" && cluster.CPUArchitecture != params.InfraenvCreateParams.CPUArchitecture {
+	if cluster != nil && cluster.CPUArchitecture != "" && cluster.CPUArchitecture != common.MultiCPUArchitecture && cluster.CPUArchitecture != params.InfraenvCreateParams.CPUArchitecture {
```

Comment:
Maybe worth some short comment here for why we skip the validation for MultiCPUArchitecture.

Reply:
Sure 👍🏻